recipes/hipace/meta.yaml

```
@@ -0,0 +1,82 @@
+{% set name = "hipace" %}
+{% set version = "23.03" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Hi-PACE/hipace/archive/v{{ version }}.tar.gz
+  sha256: f32104af1e7ca33b7ef8a3fd7964ee62db13a68a00af305d4ace5d0e86e2efb2
+
+build:
+  number: 0
```

```suggestion
build:
  # Prevent conda-build from deciding to build for each python version when hipace doesn't depend on python
  skip: True  # [py != 310]
  number: 0
```
I'm pretty sure from our discussions that hipace does NOT depend on Python, but I think some bug in conda-build is causing it to decide to build your recipe three times. Use skip here to prevent building for multiple python versions since it is not needed.